packages/eslint-plugin/tools/generate-configs.ts

```diff
@@ -19,7 +19,10 @@ interface LinterConfig extends TSESLint.Linter.Config {
 }
 
 const RULE_NAME_PREFIX = '@typescript-eslint/';
-const MAX_RULE_NAME_LENGTH = 32;
+const MAX_RULE_NAME_LENGTH = Object.keys(rules).reduce(
```

Comment:
what do you think about dropping this all together, and letting prettier format this table?

Reply:
this isn't the rule table - this formats the console output for the config generation script so that it's easier to scan:

![image](https://user-images.githubusercontent.com/7462525/72657438-1ecf5c00-3959-11ea-8ef8-658c68e4136b.png)